modules/core/src/main/java/io/fluo/recipes/common/RowRange.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2015 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.recipes.common;
+
+import com.google.common.base.Preconditions;
+import io.fluo.api.data.Bytes;
+
+public class RowRange {
```

Comment:
I thought about that.   The reason I went this is that only a row range can be compacted.    Span and Accumulo's range allow specification of more fine grained ranges that can not be compacted.